bitnami/mediawiki/1/debian-12/rootfs/opt/bitnami/scripts/mediawiki/setup.sh

```diff
@@ -32,7 +32,11 @@ fi
 mediawiki_validate
 
 # Update web server configuration with runtime environment (needs to happen before the initialization)
-web_server_update_app_configuration "mediawiki"
+ensure_web_server_app_configuration_exists "mediawiki" --type php --apache-extra-directory-configuration "
+RewriteEngine On
+RewriteRule ^/?${MEDIAWIKI_WIKI_PREFIX:1}(/.*)?$ %{DOCUMENT_ROOT}/index.php
+RewriteRule ^/?${MEDIAWIKI_SCRIPT_PATH:1}(.*)$ %{DOCUMENT_ROOT}/\$1 [L]
```

Comment:
Are `${MEDIAWIKI_WIKI_PREFIX:1}` and `${MEDIAWIKI_SCRIPT_PATH:1}` typos?

What would be the reason to slice the first character?

Reply:
the first character is the forward slash `/`, which is already present in the regexp

Reply:
Ah, that's true, I didn't notice the forward slash, thank you!